site/content/en/docs/Guides/Client SDKs/unreal.md

```diff
@@ -116,6 +116,25 @@ bDisableAutoConnect=true
 
 ## SDK Functionality
 
+| Area                 | Action                   | Implemented                   |
```

Comment:
🤔 should this be at the top?

Reply:
Top of the article or top of the section?

Reply:
Top of the section - sorry, wasn't clear. Just figure it might be information people want straight away?

Reply:
Cool, will move as I fix the tables... Monday Dom issue 😜